File: output/llama_4_maverick/avatar/direct_translation/python/java/successful_translation/atcoder_ABC164_A.java

import java.util.Scanner;

public class atcoder_ABC164_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int S = sc.nextInt();
        int W = sc.nextInt();
        System.out.println(S <= W ? "unsafe" : "safe");
    }
}

File: output/deepseek_chat/avatar/algo_based_translation/java/python/atcoder_ABC164_A.py
import sys

class atcoder_ABC164_A:
    @staticmethod
    def main(args):
        try:
            scList = []
            while True:
                try:
                    line = input()
                    scList.append(line)
                except EOFError:
                    break
            
            numlist = scList[0].split()
            Sheep = int(numlist[0])
            Wolve = int(numlist[1])
            
            if Wolve >= Sheep:
                print("unsafe")
            else:
                print("safe")
                
        except Exception as e:
            pass

if __name__ == "__main__":
    atcoder_ABC164_A.main(sys.argv)
